server/routerlicious/packages/services/src/mongodb.ts

```diff
@@ -13,6 +13,46 @@ import { MongoErrorRetryAnalyzer } from "./mongoExceptionRetryRules";
 const MaxFetchSize = 2000;
 const MaxRetryAttempts = 3;
 const InitialRetryIntervalInMs = 1000;
+const errorSanitizationMessage = "REDACTED";
+const errorResponseKeysAllowList = [
+    "_id",
+    "clientId",
+    "clientSequenceNumber",
+    "code",
+    "codeName",
+    "documentId",
+    "driver",
+    "err",
+    "errmsg",
+    "errorDetails",
+    "errorLabels",
+    "expHash1",
+    "index",
+    "insertedIds",
+    "message",
+    "minimumSequenceNumber",
+    "mongoTimestamp",
+    "name",
+    "nInserted",
+    "nMatched",
+    "nModified",
+    "nRemoved",
+    "nUpserted",
+    "ok",
+    "op",
```

Comment:
Isn't `op` is the error of customer content?

Reply:
Will remove.